kolibri/core/assets/src/views/auth-message.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+
+  <div class="auth-message">
+    <h1>
+      {{ header }}
+    </h1>
+    <p>
+      {{ details || defaultDetails }}
+    </p>
+  </div>
+
+</template>
+
+
+<script>
+
+  const userRoles = [
+    'admin',
+    'adminOrCoach',
+    'deviceOwner',
+    'learner',
+    'registeredUser'
+  ];
+
+  module.exports = {
+    props: {
+      authorizedRole: {
+        type: String,
+        validator(role) {
+          return userRoles.includes(role);
+        },
+        default: 'registeredUser',
+      },
+      header: {
+        type: String,
+        default() {
+          return this.$tr('forgetToSignIn');
+        },
+      },
+      details: { type: String },
```

Comment:
Any reason you went away from the pattern of using `default` above? I've tried 
```
details: {
        type: String,
        default() {
          return this.$tr('mustBeSignedInAsRole', { role: this.$tr(this.authorizedRole) });
        },
      },
```
locally and it seems to be doing alright